### DISCUSSION TITLE
Failed to get http url content

### DISCUSSION THREAD

**Discussion by vnishukov** [aiogram/aiogram#721] [Q&A]
Trying to send a video using regular youtube source link for stream, and get the error:
`aiogram.utils.exceptions.InvalidHTTPUrlContent: Failed to get http url content`

The link:
`https://r6---sn-jvhnu5g-bvwl.googlevideo.com/videoplayback?expire=1633631738&ei=muleYdrkOKSS8wKC3I7QBQ&ip=<MY_IP_ADDRESS_HERE>&id=o-AFJ17wnpPzEeqd-xfT0Io5WsJOFR3gaeBrZZHZSl3zlU&itag=22&source=youtube&requiressl=yes&mh=H1&mm=31,29&mn=sn-jvhnu5g-bvwl,sn-jvhnu5g-n8v6&ms=au,rdu&mv=m&mvi=6&pl=24&gcr=ru&initcwndbps=1880000&vprv=1&mime=video%2Fmp4&cnr=14&ratebypass=yes&dur=468.903&lmt=1633459703254111&mt=1633609737&fvip=6&fexp=24001373,24007246&c=ANDROID&txp=5432434&sparams=expire,ei,ip,id,itag,source,requiressl,gcr,vprv,mime,cnr,ratebypass,dur,lmt&sig=AOq0QJ8wRgIhAP93Wm4cDLNxgLQVlOPRbwl529Rivgicd2uNeyOKYVEtAiEA9MkKMpgSsq6TJi56AxkMS_qElS6wgM5PPIHwzG19758%3D&lsparams=mh,mm,mn,ms,mv,mvi,pl,initcwndbps&lsig=AG3C_xAwRQIhALMDFx1tMxObaHlgid1uRpiDqQ-ufD4MA-CUzB2ROGKsAiAW35MLEcr6AvVObCgFyMrbFx9tDZDP49E56nvXKN1sZA%3D%3D`

This link is opens by browser and working just fine.

PS
It works well for instagram links like that. What the restriction? length of link or something else? Cannot figure out

Could someone please help or explain.
Thanks!

**Comment by Olegt0rr** [comment]
Cause it's not a direct link for video file, it's a stream.

**Reply by vnishukov** [reply]
Is any workaround for this? To make it video like without downloading?

**Reply by Olegt0rr** [reply]
No
